includes/admin-page.php

```diff
@@ -0,0 +1,65 @@
+<?php 
+
+function frontity_embedded_register_menu() {
+  add_options_page(
+    'Frontity Embedded Mode',
+    'Frontity Embedded Mode',
+    'manage_options',
+    'frontity-embedded-plugin',
+    'frontity_embedded_render_admin_page'
+  );
+}
+add_action( 'admin_menu', 'frontity_embedded_register_menu' );
+
+function frontity_embedded_render_admin_page() {
+  ?>
+    <div class="wrap">
+      <h2>Frontity Embedded Mode</h2>  
+      <form method="POST" action="options.php">  
+          <?php 
+              settings_fields( 'frontity_embedded_plugin_settings' );
+              do_settings_sections( 'frontity_embedded_plugin_page' ); 
+          ?>             
+          <?php submit_button(); ?>  
+      </form> 
+    </div>
+  <?php
+}
+
+function frontity_embedded_register_my_setting() {
```

Comment:
I guess we should change the name of this function, right? :smile:

Something like

```suggestion
function frontity_embedded_register_settings() {
```

Reply:
Changed 👍